enterprise/coderd/coderdenttest/proxytest.go

```diff
@@ -131,14 +144,14 @@ func NewWorkspaceProxy(t *testing.T, coderdAPI *coderd.API, owner *codersdk.Clie
 		Tracing:           coderdAPI.TracerProvider,
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line 137]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->we _really_ need `.With(slog.F("server_url", serverURL.String())` or debugging a failure in the mesh test is going to be a nightmare<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/coder/coder/12220?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->